test/cmocka/memory_mock.x.in

```diff
@@ -6,5 +6,7 @@ SECTIONS
       *(*.static_log*)
     }
 
+    _comp_init_start = .;
```

Comment:
Is it needed here? I see that `DECLARE_COMPONENT` is doing nothing for unit tests so`.com_init` section is not used

Reply:
Yes, it is needed. Without that, component.c compilation will result in undefined reference for this two variables.